website/docs/r/dns_linked_zone.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "DNS Services"
+layout: "ibm"
+page_title: "IBM : dns_linked_zone"
+description: |-
+  Manages IBM Private DNS linked zone.
+---
+
+# ibm_dns_linked_zone
+
+The DNS linked zone resource allows users to request and manage linked zones. 
+
+
+## Example usage
+
+```
+resource "ibm_dns_linked_zone" "test" {
+  name          = "test_dns_linked_zone"
+  instance_id   = "****************************"
```

Comment:
Instead of ********* can we change to use interpolation syntax refere other dns resource docs

Reply:
For instance_id yes we can, but we can't for the owner values as these will exist in a different IBM cloud account